aquality-tracking-cucumber4-jvm/src/test/java/aquality/tracking/integrations/cucumber4jvm/integration/features/Test.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Test feature
+
+  @Demo
+  Scenario: Test passes
+    Given I pass this scenario
+
+  @Smoke @Priority1
+  Scenario Outline: Test with the same names
+    Given I pass this scenario with parameter <example value>
+
+    Examples:
+      | example value |
+      | value1        |
+
+  @Demo @Priority1
+  Scenario Outline: Test with the same names
+    Given I pass this scenario with parameter <example value>
+
+    Examples:
```

Comment:
@mialeska , it doesn't work with tags on examples.
Btw, how do you think, is it ok to use indices in names? Maybe it makes sense to use values from examples as in SpecFlow library? Because when we have several tables of examples and want to add one more example the indices will be inconsistent,

Reply:
Fixed. Now it supports tags on all available levels